<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PeriodEditor</class>
 <widget class="QSplitter" name="PeriodEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Period</string>
  </property>
  <property name="orientation">
   <enum>Qt::Vertical</enum>
  </property>
  <property name="handleWidth">
   <number>1</number>
  </property>
  <widget class="QSplitter" name="Splitter">
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="handleWidth">
    <number>1</number>
   </property>
   <widget class="QComboBox" name="TimeType">
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>28</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>28</height>
     </size>
    </property>
    <property name="toolTip">
     <string>Calendar type</string>
    </property>
    <property name="statusTip">
     <string>Calendar type</string>
    </property>
    <property name="whatsThis">
     <string>Calendar type</string>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="PeriodType">
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>28</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>28</height>
     </size>
    </property>
    <property name="toolTip">
     <string>Time period type</string>
    </property>
    <property name="statusTip">
     <string>Time period type</string>
    </property>
    <property name="whatsThis">
     <string>Time period type</string>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="TimeZone">
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>28</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>28</height>
     </size>
    </property>
    <property name="toolTip">
     <string>Time zone</string>
    </property>
    <property name="statusTip">
     <string>Time zone</string>
    </property>
    <property name="whatsThis">
     <string>Time zone</string>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="Stacked">
   <widget class="QWidget" name="Period">
    <widget class="QLabel" name="labelStart">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>140</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Start time : </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelFinish">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>140</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Finish time : </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Start">
    <widget class="QLabel" name="labelStartTime">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>140</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Start time : </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Finish">
    <widget class="QLabel" name="labelFinishTime">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>140</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Finish time : </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Spot">
    <widget class="QLabel" name="labelSpot">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>140</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Time spot : </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>PeriodType</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>PeriodEditor</receiver>
   <slot>PageChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>300</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TimeType</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>PeriodEditor</receiver>
   <slot>PageChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TimeZone</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>PeriodEditor</receiver>
   <slot>TimeZoneChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>366</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
